invenio_oaiserver/percolator.py

```diff
@@ -64,22 +67,29 @@ def _percolate_query(index, doc_type, percolator_doc_type, document):
     index = _build_percolator_index_name(index)
     if ES_VERSION[0] in (2, 5):
         results = current_search_client.percolate(
-            index=index, doc_type=doc_type, allow_no_indices=True,
-            ignore_unavailable=True, body={'doc': document}
+            index=index,
```

Comment:
same, mostly for the whole file... as much as I am inclined to use formatters... this messes with git history and might be problematic to trace changes in the future... seems to happen in the whole file...

Reply:
should be avoided in the future but not worth the time now